genie-common/src/main/java/com/netflix/genie/common/dto/Cluster.java

```diff
@@ -37,6 +40,7 @@
 
     @NotNull(message = "A valid cluster status is required")
     private final ClusterStatus status;
+    private final Set<String> dependencies = new HashSet<>();
```

Comment:
As discussed offline, postponing this refactoring since it requires changes to Command